### ISSUE TITLE
How to run the training cycle for more than 200 Epochs? 

### ISSUE THREAD

**Issue by AlexMackay94** [NVIDIA/pix2pixHD#307]
Hello! 

I cannot find the code to run the training cycle for more than 200 epochs. Is this in the base options, training options, or elsewhere? 

Thank you.

**Comment by AlexMackay94** [comment]
It was pointed out that the code for this was within train_options.py. 

` self.parser.add_argument('--niter', type=int, default=100, help='# of iter at starting learning rate')`

I added `--niter 200` to my training code and the process ran for 400 rather than 200 epochs.
